curvefs/src/client/fuse_client.cpp

```diff
@@ -580,7 +580,7 @@ CURVEFS_ERROR FuseClient::UpdateParentInodeMCTimeAndInvalidNlink(
         parentInodeWrapper->UpdateTimestampLocked(kModifyTime | kChangeTime);
 
         if (FsFileType::TYPE_DIRECTORY == type) {
-            parentInodeWrapper->InvalidateNlink();
+            parentInodeWrapper->UpdateNlinkLocked(nlink);
```

Comment:
Is there a bug here? we changed the timestamp and nlink of an inode, but we don't call `ShipToFlush` which means the updated fields don't persist to metaserver?

Reply:
The update of nlink is done in metaserver and client at same time, the nlink in client is just to show, it need refresh first from metaserver when update to metaserver.

The update of time will set inode to dirty and flush backgroud